META.md

```diff
@@ -23,3 +23,9 @@ Even though we are not using a thenable, `Promise` is a more recognizable and ac
 ## Creation of Promises
```

Comment:
I'd just rename this section to `Resolution of Promises` and remove the below header, so combine it into one section.